suites/pacific/rados/tier-3_rados_test-location-stretch-mode.yaml

```diff
@@ -130,7 +131,9 @@ tests:
       config:
         command: add
         id: client.1                      # client Id (<type>.<Id>)
-        node: node8                       # client node
+        nodes:
+          - node8:
+              release: 5                    # client node
```

Comment:
Is this to tackle the SELinux issue?

Reply:
Yes, This changes were made to tackle SELinux issues that come with the nightly builds.